config/200-clusterrole-namespaced.yaml

```
@@ -0,0 +1,55 @@
+# Copyright 2022 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: triggermesh-namespaced-admin
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
+  labels:
```

According to the docs, this applies only to **default clusterRoles**.
These roles are custom, so the annotation is superfluous / potentially confusing.

```suggestion
  name: triggermesh-namespaced-admin
  labels:
```